### DISCUSSION TITLE
tombol buka undangannya tidak bisa dipencet

### DISCUSSION THREAD

**Discussion by azrieirham** [parta99/parta99.github.io#4] [General]
masalahnya dimana yah mas
![Uploading Screenshot (55).png…]()

**Comment by azrieirham** [comment]
`[plugin:vite:import-analysis] Failed to resolve import "./TheReservasi.vue" from "src/views/LandingPage.vue". Does the file exist?
C:/Users/Admin/Downloads/pawiwahan/thema-2-VueJS/src/views/LandingPage.vue:8:28
19 |  import Carousel from "./Carousel.vue";
20 |  import TheGalery from "./TheGalery.vue";
21 |  import TheReservasi from "./TheReservasi.vue";
   |                                  ^
22 |  import TheAuthComments from "@/components/TheAuthComments.vue";
23 |`

**Reply by parta99** [reply]
gagal menemukan atau mengimpor file TheReservasi.vue yang dirujuk di dalam file LandingPage.vue.
